dev-util/clion/clion-2017.2.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils versionator
+
+SLOT="0"
+PV_STRING="$(get_version_component_range 4-6)"
```

Comment:
Where is this used? I think nowhere, plus it would be null right now. So I think you can drop all the version magic and just use `${PV}` and `${P}` appropriately all over the place.